src/PokemonGymOwner.java

```diff
@@ -0,0 +1,27 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class PokemonGymOwner {
```

Comment:
Deze klasse mag PokemonTrainer extenden. Dan hoef je in deze klasse ook niet "name" en "pokemons" opnieuw te definieren. Die kun je dan overerfen van PokemonTrainer